core/src/mindustry/entities/bullet/BulletType.java

```diff
@@ -301,6 +301,14 @@ public BulletType copy(){
         }
     }
 
+    @Override
+    public void load(){
+        for(var part : parts){
+            part.turretShading = false;
```

Comment:
Should this even be here? Should bullet parts be hardcoded into unit shading or allow for `turretShading` to be set manually?

Reply:
No turret shading, it's nonsense for bullets.